iOSClient/Main/Collection Common/NCCollectionViewCommon.swift

```diff
@@ -154,6 +154,13 @@ class NCCollectionViewCommon: UIViewController, UIGestureRecognizerDelegate, UIS
         preferences.animating.dismissDuration = 1.5
 
         tipView = EasyTipView(text: NSLocalizedString("_tip_accountrequest_", comment: ""), preferences: preferences, delegate: self)
+        if(!UserDefaults.standard.bool(forKey: "isInitialPrivacySettingsShowed") || isApplicationUpdated()){
```

Comment:
isInitialPrivacySettingsShowed key can be extracted into Constant and then use that Constant variable everywhere to avoid any issues.